frontend/lib/src/components/widgets/DataFrame/DataFrame.test.tsx

```diff
@@ -29,6 +29,10 @@ jest.mock("@glideapps/glide-data-grid", () => ({
   DataEditor: jest.fn(props => <div {...props} />),
 }))
 
+// The native-file-system-adapter creates some issues in the test environment
+// so we mock it out:
+jest.mock("native-file-system-adapter", () => ({}))
```

Comment:
What's the issue that it causes?

Reply:
I added a bit more info in a comment. I'm not 100% sure what causes the issue, but I'm quite certain that the saveFilePicker would anyways not be available in jest-dom...